airbyte-integrations/connectors/source-orb/source_orb/source.py

```diff
@@ -268,6 +275,13 @@ def transform_record(self, ledger_entry_record):
         del ledger_entry_record["customer"]
         ledger_entry_record["customer_id"] = nested_customer_id
 
+        # Un-nest credit_block -> expiry_date into block_expiry_date and per_unit_cost_basis
+        nested_expiry_date = ledger_entry_record["credit_block"]["expiry_date"]
+        nested_per_unit_cost_basis = ledger_entry_record["credit_block"]["per_unit_cost_basis"]
+        del ledger_entry_record["credit_block"]
+        ledger_entry_record["block_expiry_date"] = nested_expiry_date
+        ledger_entry_record["credit_block_per_unit_cost_basis"] = nested_per_unit_cost_basis
+
         return ledger_entry_record
```

Comment:
@anushree-agrawal would be nice to have a unit test to validate the data transformation here. wydt?

Reply:
Sure, I'll add one!